scripts/yaml_characters

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bin/rails runner
+# frozen_string_literal: true
+
```

Comment:
let's add a quick explanation of what this does? Or add it as `--help` / default banner text in optparse?

```suggestion
# Script that prints all unique characters across user-facing text, that can be used to pare down font glyphs
```

Reply:
Makes sense 👍 Added banner / `--help` in ec92027.